### PR TITLE
Prevent the school card from being empty.

### PR DESCRIPTION
**Pull request by aikewoody**
# Description

_Description of the feature or fix with link to the issue if existing_

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

_Please attach screenshots showing the change if relevant_

## Checklist

- [ ] I have updated mockServer
- [ ] I've added analytics as relevant
- [ ] I have run `npm test`
- [ ] I have run `npm run test:i18n`

## Out of scope and potential follow-up

_Is there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?_

### CODE REVIEW

File: src/features/dashboard/DashboardScreen.tsx [hunk 1]
```diff
@@ -171,15 +169,7 @@ export function DashboardScreen({ navigation, route }: IProps) {
...
-        {hasNetworkData ? (
-          <View
-            style={{
-              marginVertical: 8,
-            }}
-          >
-            <SchoolNetworks schoolGroups={networks!} />
-          </View>
-        ) : null}
+        <SchoolNetworks schoolGroups={schoolGroups} style={styles.marginVertical} />
```
Comment (juliatan) [inline src/features/dashboard/DashboardScreen.tsx:172]: Don't know what best practice is (I see both throughout our code base), but do we really need to define this below? Perhaps inline styles suffices if we're going to call it `marginVertical`.

File: src/components/cards/school-networks/index.tsx [hunk 2]
```diff
@@ -5,13 +5,13 @@ import { StyleProp, ViewStyle } from 'react-native';
 import Card from './Card';
 import { SContainerView } from './styles';
 
-type Props = {
+type TProps = {
```
Comment (juliatan) [inline src/components/cards/school-networks/index.tsx:8]: Also not sure about adding `T` or `I` in front of our types and interfaces lol. Do we really need to do this? I guess it's personal preference - is this what we've settled on throughout our code base?

File: src/features/dashboard/DashboardScreen.tsx (hunk 1 above)
Comment (selvach) [reply src/features/dashboard/DashboardScreen.tsx:172]: My personal preference is to have const styles = StyleSheet.create({}) for any screen specific adjustments. 
I haven't fully internalised StyledComponents. In our list of things to discuss and address: 

https://www.notion.so/joinzoe/2021-App-Spring-Cleaning-61039ea8a425483bbe95ef99c30b1af1#4c8f28087b2f406795ad7959b7ff1338

**Review by juliatan — APPROVED** [review]

**Review by selvach — APPROVED** [review]

File: src/components/cards/school-networks/index.tsx (hunk 2 above)
Comment (selvach) [reply src/components/cards/school-networks/index.tsx:8]: Yes! Could you start a style guide with @aikewoody (checked into the repo, please).

**Pull request merged**